transform-test-selector-params-to-hash-pairs.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env node */
+var TEST_SELECTOR_PREFIX = /data-test-.*/;
```

Comment:
We have such a regex already in `addon/utils/bind-data-test-attributes.js`. I think it makes sense to refactor this into a shared function like `isTestSelector`. Would that make sense?